src/Workspaces/Core/Portable/CodeFixes/FixAllOccurrences/BatchFixAllProvider.cs

```diff
@@ -24,187 +22,227 @@ namespace Microsoft.CodeAnalysis.CodeFixes
     /// <summary>
     /// Helper class for "Fix all occurrences" code fix providers.
     /// </summary>
-    internal partial class BatchFixAllProvider : FixAllProvider
+    internal sealed class BatchFixAllProvider : AbstractDefaultFixAllProvider
```

Comment:
this type now derives from AbstractDefaultFixAllProvider.  Taht new type contains the logic for processing the solutoin in an appropriate order, and unifying the logic for fix-all-in-doc/proj.solution.

Reply:
this type is also significantly rewritten.  i recommend just looking at the 'after' logic for the new way that batch fixing works.